doc/dynamo0p3.rst

```diff
@@ -1004,6 +1004,21 @@ does not necessarily reflect the actual implementation of the
 built-in (*e.g.* it could be implemented by PSyclone
 generating a call to an optimised maths library).
 
+axmy
```

Comment:
Please add "Modified I. Kavcic Met Office " at the appropriate place as you have done for other files.